### ISSUE TITLE
Action links (Comment, Like, ...) not reachable with TAB key

### ISSUE THREAD

**Issue by taivolints** [pepyatka/pepyatka-html#388]
When using the TAB key (on keyboard) to move keyboard focus sequentially over the links on the page, the data-ember-action links such as Comment, Like, Hide, Edit, NN other people, etc., cannot be reached, that is, they are being skipped over, i.e., they are not tab navigable.
